src/Raven.Client/ServerWide/DatabaseRecord.cs

```diff
@@ -453,6 +454,8 @@ public class IndexHistoryEntry
         
         public DateTime CreatedAt { get; set; }
         
+        public string CertificateThumbprint { get; set; }
```

Comment:
The problem is that this is just meaningless numbers.
Better to store the thumpbrint & subject. Which likely have more information.

Can just call this "Additional Info"

Reply:
Done, I discussed the topic with @arekpalinski, and we decided to concatenate certificate subject and thumbprint with source